signer/tests/integration/emily.rs

```diff
@@ -461,6 +461,7 @@ async fn deposit_flow() {
         signing_round_max_duration: Duration::from_secs(10),
         dkg_max_duration: Duration::from_secs(10),
         sbtc_contracts_deployed: true,
+        is_epoch3: true,
```

Comment:
we really should have a `new()` method so that we don't have to refactor all instantiations when a new field is added... but yeah, another day